packages/esm-home-app/src/homepage-header-date/homepage-header-date.scss

```diff
@@ -0,0 +1,3 @@
+.value {
```

Comment:
The contents of this file are exact to `packages/esm-home-app/src/header-date/header-date.scss` and it seems only one of them is being imported. Please harmonise this and delete one of them

Reply:
Thanks @pirupius , this is done.